### DISCUSSION TITLE
TextInput does not add mask in firefox

### DISCUSSION THREAD

**Discussion by Maagalhaes** [filamentphp/filament#7844] [General]
Hello!

Filament v3

When I create a TextInput with mask and use firefox it only adds the mask after the blur, whereas in chrome it adds it as I type.

Using the firefox developer I noticed that the function of the livewire.js file:

```js
function flushJobs() {
    flushPending = false;
    flushing = true;
    for (let i = 0; i < queue.length; i++) {
      queue[i]();
      lastFlushedIndex = i;
    }
    queue.length = 0;
    lastFlushedIndex = -1;
    flushing = false;
  }
```

In chrome when you run 'queue' the mask is kept, in firefox the mask is lost.
In this case the queue is executing only the reactiveEffect.

Sorry for the bad english.

**Comment by pboivin** [comment]
I could be wrong but this looks like a Livewire (or Alpine.js) issue, not directly related to Filament...

**Reply by Maagalhaes** [reply]
I created a new laravel project and installed livewire 3 without the filament and the mask worked.
